server/views/serviceProviderReferrals/sessions/feedback/confirmationAllSessionsCompleted.njk

```diff
@@ -1,12 +1,66 @@
 {% extends "../../../partials/layout.njk" %}
+{% set mainClasses = "govuk-main-wrapper--l" %}
```

Comment:
What does this do? I can't seem to find where it's used

Reply:
This is part of the confirmation page pattern. Please have a look: https://design-system.service.gov.uk/patterns/confirmation-pages/

Reply:
Ah sorry, it overrides the default template's CSS class.